pkg/aspect/root/flags/bazel_flags.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright © 2021 Aspect Build Systems Inc
+
+Not licensed for re-use.
+*/
+
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"aspect.build/cli/pkg/bazel"
+)
+
+var (
+	exposedBazelFlags = []string{
+		"keep_going",
+		"expunge",
+		"expunge_async",
+		"show_make_env",
+	}
+)
+
+type MultiString struct {
+	value *[]string
```

Comment:
Why make it a pointer if you are always dereferencing it over and over in the methods?